css/redirector.css

```diff
@@ -479,7 +479,7 @@ a.disabled:hover, button[disabled]:hover {
 }
 
 .hidden {
-  display:none;
+  display:none !important;
```

Comment:
Does this CSS ruleset really need `!important` added?  Is there a way to avoid adding `!important`?

Reply:
`.form-grid > div` selector has a higher specificity and cause an issue so I thought as a utility class it's ok to add `!important`
but It can be avoided by adding the same declaration with  `.replace-process-input.hidden` selector.

Reply:
Thanks.  I'm good either way.  I think adding the `.replace-process-input.hidden` selector to the `.hidden` ruleset is a slightly better option.  It's a bit cleaner and will prevent any problems in the future if someone removes `!important` from `.hidden` because it's not documented why it's present.